src/components/list/list-renderer.tsx

```diff
@@ -143,13 +144,25 @@ export class ListRenderer {
                 {...attributes}
             >
                 {item.icon ? this.renderIcon(this.config, item) : null}
+                {this.getCustomComponent(item)}
```

Comment:
I thought this was supposed to be used instead of a regular icon :thinking: Should you be able to specify _both_ an icon and a component?

Reply:
Answered in a previous thread